scripts/lib/get_tests.py

```diff
@@ -65,7 +65,7 @@
                              "ERR.f45_g37_rx1.A",
                              "ERP.f45_g37_rx1.A",
                              "SMS_D_Ln9.f19_g16_rx1.A",
-                             "DAE.f19_f19.A",
+                             "DAE.f19_f19.ADWAV",
```

Comment:
@jgfouca, I requested your review because this represents a change to the tests run by scripts_regression_tests.py

Reply:
Shouldn't be a problem.